src/server/models/Job.js

```diff
@@ -0,0 +1,29 @@
+const mongoose = require('mongoose');
+
+const JobsSchema = new mongoose.Schema({
+    job_id: {
+        type: Number,
+        required: true,
+        unique: true
+    },
+    description: {
+        type: String,
+        required: true
+    },
+    salary: {
+        type: Number,
+        required: true
+    },
+    company: {
+        type: String,
+        required: true
+    },
+    category: {
+        type: String,
+        required: true
+    }
+});
```

Comment:
We should add job title in the schema, as many recruiters mention the job title in their job posting on LinkedIn, Indeed, ...